src/pages/Auth/FindIDPassword/page/VerifyCode.tsx

```diff
@@ -1,55 +1,20 @@
-import { UseFormRegister, UseFormHandleSubmit } from 'react-hook-form';
 import { useState } from 'react';
 import { useParams, useNavigate } from 'react-router-dom';
 import cn from 'utils/ts/classNames';
+import { sendEmail, getAccount } from '../api';
 import Modal from '../component/Modal';
 import useInputCheck from '../hook/useInputCheck';
 import style from './VerifyCode.module.scss';
-
-interface PropData {
-  register: UseFormRegister<FormData>,
-  handleSubmit: UseFormHandleSubmit<FormData>,
-}
-
-interface InputInfo {
-  register: UseFormRegister<FormData>,
-  name: 'first' | 'second' | 'third' | 'fourth',
-  inputRef: React.MutableRefObject<HTMLInputElement[] | null[]>,
-  preventOverLength: (e: React.ChangeEvent<HTMLInputElement>, next: number) => void,
-  n: number,
-  index: number,
-}
-
-export interface FormData {
-  first: string,
-  second: string,
-  third: string,
-  fourth: string
-}
-
-function Input({
-  register, name, inputRef, preventOverLength, n, index,
-}: InputInfo) {
-  const inputRefCopy = inputRef;
-  return (
-    <input
-      type="number"
-      className={cn({ [style['form__input--block']]: true })}
-      {...register(name, {
-        required: true,
-        maxLength: 1,
-      })}
-      ref={(e) => { register(name).ref(e); inputRefCopy.current[index] = e; }}
-      onChange={(e) => preventOverLength(e, n)}
-    />
-  );
-}
+import { RegisterProp, CodeInfo } from '../entity';
+import Input from '../component/Input';
 
 const NAME: ['first', 'second', 'third', 'fourth'] = ['first', 'second', 'third', 'fourth'];
```

Comment:
`as const`를 키워드를 활용해 타입을 완전상수화 시킬 수 있습니다.
근데 NAME이라는 이름이 어울리나 싶긴 하네요?
```suggestion
const NAME = ['first', 'second', 'third', 'fourth'] as const;
```

Reply:
CODE라는 이름으로 완전상수화 시켰습니다.
좋은 정보 감사합니다